presto-docs/src/main/sphinx/sql/call.rst

```diff
@@ -28,3 +28,7 @@ Call a procedure using named arguments::
 Call a procedure using a fully qualified name::
 
     CALL catalog.schema.test();
+
+.. note::
+    CALL statement is NOT used to invoke a DBMS style stored procedure from the underlying
```

Comment:
How about something like
```
Procedures can be defined by connectors to perform data manipulation or administrative tasks.
For example, the :doc:`/connector/system` connector defines a procedure for killing a running query.
Some connectors, such as the :doc:`/connector/postgresql`, are for systems that have
their own stored procedures. These stored procedures are separate from the connector-defined
procedures discussed here and thus are not directly callable via ``CALL``. See connector
documentation for details on available procedures.
```

This could be moved to the description, rather than being a note.